### DISCUSSION TITLE
2 Relays with opossite states how?

### DISCUSSION THREAD

**Discussion by mmmmm009** [arendst/Tasmota#20837] [Q&A]
Hi,

Using Tasmota scripting I'd like to set 2 relays to act opposite - opposite states.

;on Power changer
`>E
pwr[2] XXX = pwr[1]`

is there any opperator doing this?

**Comment by sfromis** [comment]
https://tasmota.github.io/docs/Commands/#interlock operates independently of scripting

**Comment by mmmmm009** [comment]
hmm..
Not sure if It does what I ment.

When I do Power1=ON then Power2 should go OFF.
When Power2=ON then Power1=OFF

**Reply by sfromis** [reply]
Not sure that you explained what you meant then. Interlock **is** opposite states, will turn off the other relay when you turn one of them on.

**Comment by mmmmm009** [comment]
So..
In my module configuration I have 4 Relays and 3 PWM channels.
All together POWER1....POWER7 (7 buttons on the main page)

```
15:04:49.465 RSL: RESULT = {"POWER1":"ON"}
15:04:49.468 RSL: POWER1 = ON
15:04:51.446 RSL: RESULT = {"POWER2":"ON"}
15:04:51.449 RSL: POWER2 = ON
15:04:52.931 RSL: RESULT = {"POWER3":"ON"}
15:04:52.934 RSL: POWER3 = ON
15:04:55.070 RSL: RESULT = {"POWER4":"ON"}
15:04:55.073 RSL: POWER4 = ON
15:04:59.587 RSL: RESULT = {"POWER5":"ON"}
15:04:59.590 RSL: POWER5 = ON
15:05:04.679 RSL: RESULT = {"POWER6":"ON"}
15:05:04.682 RSL: POWER6 = ON
15:05:08.241 RSL: RESULT = {"POWER7":"ON"}
15:05:08.244 RSL: POWER7 = ON
```

I'd like to create 3 groups:
```
15:07:00.949 CMD: Interlock 2,5 3,6 4,7
15:07:00.953 RSL: RESULT = {"Interlock":"OFF","Groups":"2,5 3,6"}
```
Why is group no.3 skipped?
I would appreciate any hints.

**Comment by mmmmm009** [comment]
With 4 Relays and 3 PWMs I get the following:
```
13:41:51.615 CMD: Interlock
13:41:51.619 RSL: RESULT = {"Interlock":"OFF","Groups":"1,2,3,4,5,6"}
```

Why is 7 missing?
